modules/calc/src/main/java/com/opengamma/strata/calc/config/ImmutableMeasure.java

```diff
@@ -99,9 +119,12 @@ public String toString() {
   }
 
   private ImmutableMeasure(
-      String name) {
+      String name,
+      boolean currencyConvertible) {
     JodaBeanUtils.notNull(name, "name");
+    JodaBeanUtils.notNull(currencyConvertible, "currencyConvertible");
```

Comment:
Well you can't be too careful :)

I've got a template for bean properties which adds `notNull` and I forgot to delete it.